# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not perform DESTDIR based packaging - so we do not care about such files ...

  - Rene Rebe <rene@exactcode.de>

--- ./bin/package-ooo.vanilla	2005-04-19 10:12:23.000000000 +0200
+++ ./bin/package-ooo	2005-04-19 10:13:59.000000000 +0200
@@ -273,11 +273,6 @@
     mv $1.unique $1
 }
 
-if ! test -f $OODESTDIR/gid_Module_Root; then
-    echo "Failed to generate package file lists";
-    exit 1;
-fi
-
 cd $OODESTDIR
 
 if test -f gid_Module_Root_Files_2; then
@@ -446,9 +451,6 @@
 	find . -path './pkg' -prune -o -not -name 'gid_Module_*' -not -type d -exec echo "File not packaged: {}" \;
 fi
 
-echo "Cleaning up lists of files...";
-mv -f $OODESTDIR/gid_Module_* $BUILDDIR
-
 cd $BUILDDIR
 
 # move English help to an extra list
